extensions/spotify-player/package.json

```
@@ -2,7 +2,7 @@
   "$schema": "https://www.raycast.com/schemas/extension.json",
   "name": "spotify-player",
   "title": "Spotify Player",
-  "description": "Search and Play Spotify music from the Raycast, control your library with convenient commands like Like",
+  "description": "Search and Play Spotify music from Raycast and control your library with convenient commands.",
```

That reads weirdly, I think you need a comma
```suggestion
  "description": "Search and Play Spotify music from Raycast, and control your library with convenient commands.",
```